extensions/editor.js

```diff
@@ -3,3 +3,17 @@
  */
 import './shared/public-path';
 import './shared/block-category';
+import analytics from '../_inc/client/lib/analytics';
```

Comment:
Just to highlight it again, this means we'll have to add those files (and their deps -- fortunately only one leaf dep!) to the WP.com side patch (D30176-code), since they're currently not there.